packages/neuron-wallet/tests/setup.ts

```diff
@@ -1,30 +1,33 @@
 import { systemScripts } from '@nervosnetwork/ckb-sdk-utils'
 import '../src/locales/i18n'
 import { LedgerHID, LedgerCkbApp } from './mock/hardware'
-export const originalXMLHttpRequest = window.XMLHttpRequest
-
-export const mockedXMLHttpRequest = {
-  open: jest.fn(),
-  send: jest.fn(),
-  readyState: null,
-  responseText: null,
-}
-
-Object.defineProperty(mockedXMLHttpRequest, 'readyState', {
-  get: jest.fn(() => 4),
-  set: jest.fn()
-});
-Object.defineProperty(mockedXMLHttpRequest, 'responseText', {
-  get: jest.fn(() => JSON.stringify([])),
-  set: jest.fn()
-});
-
-Object.defineProperty(window, "XMLHttpRequest", jest.fn(() => ({
-  open: mockedXMLHttpRequest.open,
-  send: mockedXMLHttpRequest.send,
-  readyState: mockedXMLHttpRequest.readyState,
-  responseText: mockedXMLHttpRequest.responseText
-})));
+
+// export const mockedXMLHttpRequest = {
+//   open: jest.fn(),
+//   send: jest.fn(),
+//   readyState: null,
+//   responseText: null
+// }
+
+// Object.defineProperty(mockedXMLHttpRequest, 'readyState', {
+//   get: jest.fn(() => 4),
+//   set: jest.fn()
+// })
+// Object.defineProperty(mockedXMLHttpRequest, 'responseText', {
+//   get: jest.fn(() => JSON.stringify([])),
+//   set: jest.fn()
+// })
+
+// Object.defineProperty(
+//   window,
+//   'XMLHttpRequest',
+//   jest.fn(() => ({
+//     open: mockedXMLHttpRequest.open,
+//     send: mockedXMLHttpRequest.send,
+//     readyState: mockedXMLHttpRequest.readyState,
+//     responseText: mockedXMLHttpRequest.responseText
+//   }))
+// )
 
```

Comment:
Seems we are not using these code, and these code prevent us to use `jest-evironment node`, so I commented out

Reply:
If this, I think we should delete it but not comment it.